src/content/operations/usage/_index.en.md

```diff
@@ -0,0 +1,825 @@
+  Local Endpoint:
+    Backend:          libreswan
+    cable_name:       submariner-cable-cluster2-172-17-0-7
+    cluster_id:       cluster2
+    Health Check IP:  10.2.224.0
+    Hostname:         cluster2-worker
+    nat_enabled:      false
+    private_ip:       172.17.0.7
+    public_ip:        
+    Subnets:
+      100.2.0.0/16
+      10.2.0.0/16
+  Status Failure:  
+  Version:         v0.8.0-pre0-1-g5d7f163
+Events:            <none>
+```
+<!-- markdownlint-enable no-trailing-spaces -->
+To verify that service discovery is installed properly, validate that the Submariner CRDs have been created in the cluster:
+
+```bash
+$ kubectl get crds | grep -iE 'submariner|multicluster.x-k8s.io'
+clusters.submariner.io                    2020-11-30T13:50:33Z
+endpoints.submariner.io                   2020-11-30T13:50:33Z
+gateways.submariner.io                    2020-11-30T13:50:33Z
+servicediscoveries.submariner.io          2020-11-30T13:50:34Z
+serviceexports.lighthouse.submariner.io   2020-11-30T13:50:33Z
+serviceexports.multicluster.x-k8s.io      2020-11-30T13:50:34Z
+serviceimports.lighthouse.submariner.io   2020-11-30T13:50:33Z
+serviceimports.multicluster.x-k8s.io      2020-11-30T13:50:33Z
+submariners.submariner.io                 2020-11-30T13:50:24Z
+```
+
+Check that the `submariner-lighthouse-coredns` Service is ready:
+
+```bash
+$ kubectl -n submariner-operator get service submariner-lighthouse-coredns
+NAME                            TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)   AGE
+submariner-lighthouse-coredns   ClusterIP   100.2.177.123   <none>        53/UDP    126m
+```
+
+Verify that CoreDNS was properly configured to forward requests sent to `clusterset.local` to Lighthouse CoreDNS Server in the
+cluster:
+
+```bash
+$ kubectl -n kube-system describe configmap coredns
+Name:         coredns
+Namespace:    kube-system
+Labels:       <none>
+Annotations:  <none>
+
+Data
+====
+Corefile:
+----
+#lighthouse-start AUTO-GENERATED SECTION. DO NOT EDIT
+clusterset.local:53 {
+    forward . 100.2.177.123
+}
+#lighthouse-end
+.:53 {
+    errors
+    health {
+       lameduck 5s
+    }
+    ready
+    kubernetes cluster2.local in-addr.arpa ip6.arpa {
+       pods insecure
+       fallthrough in-addr.arpa ip6.arpa
+       ttl 30
+    }
+    prometheus :9153
+    forward . /etc/resolv.conf
+    cache 30
+    loop
+    reload
+    loadbalance
+}
+```
+
+Note that **100.2.177.123** is the ClusterIP address of the `submariner-lighthouse-coredns` Service we verified earlier.
+
+### 2. Export Services Across Clusters
+
+At this point, we have enabled secure IP communication between the connected clusters and formed the cluster set infrastructure. However,
+further configuration is required in order to signify that a Service should be visible and discoverable to other clusters in the cluster
+set. This can be done by creating a `ServiceExport` object in each cluster within the namespace that the underlying Service resides in.
```

Comment:
```suggestion
set. This can be done by creating a `ServiceExport` object in a cluster within the namespace that the underlying Service resides in.
```